sre-recipes/recipes/currencies_recipe/currencies_recipe.py

```diff
@@ -55,12 +55,24 @@ def _deploy_state(state):
         logging.info('Deleting pod: %s', delete_pod_command)
         CurrenciesRecipe._run_command(delete_pod_command)
 
+    @staticmethod
+    def _auth_cluster():
+        """ Authenticates for kubectl commands. """
+        logging.info('Authenticating cluster')
+        get_project_command = "gcloud config list --format value(core.project)"
+        project_id, error = CurrenciesRecipe._run_command(get_project_command)
+        project_id = project_id.decode("utf-8").replace('"', '')
+        auth_command = "gcloud container clusters list --filter name:cloud-ops-sandbox --project {} --format value(zone)".format(project_id)
```

Comment:
isn't it supposed to be
```bash
gcloud container clusters get-credentials
```
?

Reply:
Yep sorry missed that command :D fixed now!